data/mods/Sky_Island/items.json

```diff
@@ -521,5 +521,27 @@
     "material": [ "stone" ],
     "symbol": "y",
     "color": "green"
+  },
+  {
+    "id": "warped_autodoc_upgrader_1",
+    "type": "TOOL",
+    "name": { "str": "warped autodoc enhancer" },
+    "description": "A small device, so complex you can't entirely tell what it does.  You can see a miniature chair, a bed, some kind of machine, a computer console, a tangle of mechanical arms, all changing from moment to moment.  When used on an existing autodoc, you can upgrade it to a more advanced model.",
```

Comment:
People may not understand that _existing_ autodoc is in fact a Mk. XI model and may try to apply this thing to Mk. XXII or even Mk. XXXIII models in a vain hope that there are more advanced models (players who doesn't follow the github won't know that there are three models total). I suggest either change the name or description to clearly state which upgrader is for which model.
The same for upgrader_2 below.